src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_anubarak_trial.cpp

```diff
@@ -313,7 +314,7 @@ class boss_anubarak_trial : public CreatureScript
                     {
                         me->SetUnitFlag(UNIT_FLAG_NOT_SELECTABLE);
                         bool berserk = me->HasAura(SPELL_BERSERK);
-                        me->RemoveAllAuras();
+                        me->CastSpell(me, SPELL_CLEAR_ALL_DEBUFFS, false);
```

Comment:
DoCast()

Reply:
thanks for the repply, if you thing it's wrong, please do a change if necessary, it's my first time

Reply:
```suggestion
                        DoCastSelf(SPELL_CLEAR_ALL_DEBUFFS, true);
```

Reply:
You can change them all to this